README.md

```diff
@@ -21,24 +21,23 @@ To build these apps from source you will need to specify credentials in Gradle p
 The following secrets need to be injected into Gradle by storing them in `gradle.properties` file in
 the project root:
 
-- `ABLY_API_KEY`: On your [Ably accounts page](https://ably.com/accounts/), select your application,
-  and paste an API Key from the API Keys tab (with relevant capabilities for either subscriber/
-  publisher). This API key needs the following capabilities: `publish`, `subscribe`, `history`
-  and `presence`.
 - `MAPBOX_DOWNLOADS_TOKEN`: On
   the [Mapbox Access Tokens page](https://account.mapbox.com/access-tokens/), create a token with
   the `DOWNLOADS:READ` secret scope.
-- `MAPBOX_ACCESS_TOKEN`: On
-  the [Mapbox Access Tokens page](https://account.mapbox.com/access-tokens/), create a token with
-  public scopes.
+- `AUTHORIZATION_HEADER_BASE_64`: Base64 encoded "user:password" used to authorize with the backend service. For more details see [Ably Asset Tracking Backend Demo](https://github.com/ably/asset-tracking-backend-demo)
+- `AUTHORIZATION_USERNAME`: username used to authorize with the backend service and as clientId
```

Comment:
Password is already encoded as part of the `AUTHORIZATION_HEADER_BASE_64`, and since we don't need it in plain text, I prefer to use an encoded ready-to-use header. 
Unfortunately, we need a plaintext username to pass to the publisher as a `clientId`.

Both of those are temporary solutions, and will be removed in step 3 when users will have a way to enter their own credentials